descriptors/ModuleDescriptor-template.json

```diff
@@ -20,6 +20,33 @@
         }
       ]
     },
+    {
+      "id": "inventory-reports",
+      "version": "0.1",
+      "handlers": [
+        {
+          "methods": [
+            "GET"
+          ],
+          "pathPattern": "/circulation/inventory-reports/items-in-transit",
```

Comment:
Based upon [Emma's comment](https://issues.folio.org/browse/UIIN-840?focusedCommentId=64332&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-64332), I think we could consider using a path that is independent of circulation e.g. `/inventory-reports/items-in-transit`.

That would then make it easier to move this report implementation to a different module, either inventory or independent, in the future, if we found it fitted better there.

I think we should also consider making the interface specific only to the in transit report, as smaller interfaces can evolve more independently.